agent/app/agent_capability_unix.go

```diff
@@ -232,3 +235,14 @@ func (agent *ecsAgent) getTaskENIPluginVersionAttribute() (*ecs.Attribute, error
 func defaultIsPlatformExecSupported() (bool, error) {
 	return true, nil
 }
+
+func defaultIsGuardDutySupported() (bool, error) {
+	isEnabled, err := pathExists(config.GuardDutyServiceFilePath, false)
+	if err != nil {
+		logger.Warn("GuardDuty Capability: Cannot find GuardDuty service file path. GuardDuty capabilities will "+
```

Comment:
based on https://github.com/aws/amazon-ecs-agent/blob/b13992a87cfc5f89ce338d58c12fd7c61d4f6c83/agent/app/agent_capability.go#L575-L582

I think err !=nil implies that something other "not-exists" err was returned. this warn statement can be like

```
logger.Warn("Unable to determine if GuardDuty is installed, not reporting the capability", logger.Fieldslogger.Fields{
	field.Error: err,
})
```

Reply:
updated this